app/src/main/java/com/example/ayapp/habits/Habit.java

```diff
@@ -0,0 +1,59 @@
+package com.example.ayapp.habits;
+
+/**
+ * Created by amoughnie on 07-May-16.
+ */
+public class Habit {
+
+    private String title;// habit title
+    private String description;// habit description
+    private int goal;// number of days in which you want to preform this habit
+    private int id; // used as an index in the database
+
+    public Habit(String title,String description, int goal, int id){
+        this.title = title;
+        this.description = description;
+        this.goal = goal;
+        this.id = id;
+    }
+
+    public Habit(){
```

Comment:
Please put the default constructor above the regular constructor and add a section comment:

/***********************************************************************************************
     * CONSTRUCTORS
     **********************************************************************************************/